includes/templates/class-amp-post-template.php

```diff
@@ -321,18 +342,10 @@ private function build_post_comments_data() {
 	 * Build post content.
 	 */
 	private function build_post_content() {
-		$amp_content = new AMP_Content(
-			post_password_required( $this->post ) ? get_the_password_form( $this->post ) : $this->post->post_content,
-			amp_get_content_embed_handlers( $this->post ),
-			amp_get_content_sanitizers( $this->post ),
-			[
-				'content_max_width' => $this->get( 'content_max_width' ),
-			]
-		);
+		/** This filter is documented in wp-includes/post-template.php */
+		$content = apply_filters( 'the_content', get_the_content( null, false, $this->post ) );
```

Comment:
This introduced a regression. Prior to this change, posts having content with page breaks (`<!--nextpage-->`) would have their pages all displayed together. This needs to be restored, with a big reason being that there has been no multipage pagination in the Reader mode template. It's also not consistent with the previous behavior.

Reply:
Workaround plugin to restore the previous behavior: https://gist.github.com/westonruter/559c7a9cbb9460d1099c3d5d9bfa5ad1

Reply:
Fix: https://github.com/ampproject/amp-wp/pull/4547